README.rst

```diff
@@ -47,6 +47,7 @@ Notes
   completely without the involvement of messages. However, channel and author
   information is still available.
 * All descriptions are **required**.
+* You must grant the bot `applications.commands` permissions in the OAuth2 section of the dashboard. It is recommended to also turn on "Requires OAuth2 Code Grant" in the Bot section of the developer dashboard.
```

Comment:
```suggestion
* You must grant the bot `applications.commands` permissions in the OAuth2 section of the developer dashboard.
```

Reply:
If I do this I don't have to build it with `setup.py`?

Reply:
you don't